packages/opentelemetry/src/trace.ts

```diff
@@ -134,9 +190,12 @@ export function startInactiveSpan(options: OpenTelemetrySpanContext): Span {
 
     const spanOptions = getSpanOptions(options);
 
-    const span = tracer.startSpan(name, spanOptions, ctx);
+    if (!hasSpansEnabled()) {
```

Comment:
l: what about following to have the same return (so we don't have to touch two parts of the code in case the return changes (or a nested ternary):

```js
let context = ctx;

if (!hasSpansEnabled()) {
  context = isTracingSuppressed(ctx) ? ctx : suppressTracing(ctx);
}

return tracer.startSpan(name, spanOptions, context);
```